pkg/controller/defaults/placements.go

```diff
@@ -213,5 +214,41 @@ var (
 				},
 			},
 		},
+
+		"rgw-multi-instances": rook.Placement{
+			PodAntiAffinity: &corev1.PodAntiAffinity{
+				RequiredDuringSchedulingIgnoredDuringExecution: []corev1.PodAffinityTerm{
+					corev1.PodAffinityTerm{
+						LabelSelector: &metav1.LabelSelector{
+							MatchExpressions: []metav1.LabelSelectorRequirement{
+								metav1.LabelSelectorRequirement{
+									Key:      app,
+									Operator: metav1.LabelSelectorOpIn,
+									Values:   []string{"rook-ceph-rgw"},
+								},
+							},
+						},
+						TopologyKey: corev1.LabelHostname,
+					},
+				},
+				PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+					corev1.WeightedPodAffinityTerm{
+						Weight: 100,
+						PodAffinityTerm: corev1.PodAffinityTerm{
+							LabelSelector: &metav1.LabelSelector{
+								MatchExpressions: []metav1.LabelSelectorRequirement{
+									metav1.LabelSelectorRequirement{
+										Key:      app,
+										Operator: metav1.LabelSelectorOpIn,
+										Values:   []string{"rook-ceph-rgw"},
+									},
+								},
+							},
+							TopologyKey: "topology.kubernetes.io/zone",
```

Comment:
I couldn't use it, because in my go `k8s-api` package don't have that particular `Topology` (above L24th variable) ... Tried to update `go mod vendor` didn't see any change in the `well_known_labels.go` file

Reply:
Ohh... I wonder if it's not in the version we're vendoring...

What about this one? https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/api/core/v1/well_known_labels.go#L22 It will still be supported for a while, we can change it once the topology labels are in our vendors.

Reply:
Ok... Made the change accordingly...